eve/main.yml

```diff
@@ -280,9 +280,9 @@ stages:
       - ShellCommand:
           name: Build docker image
           command: >-
-            docker build
-            -t %(prop:docker_image_name)s
-            -t zenko/cloudserver:latest-%(prop:product_version)s
+            docker build --no-cache
```

Comment:
There's a funny thing here. You're using a worker of type `local` on this stage. This means (you guys already know) that the `ShellCommands` are executed on the same host as the buildbot/eve backends. Lucky you there's docker installed and everything is good to go for all the commands to work. But the thing is, knowing what's in there, I wouldn't recommend to do this anymore, there's so much custom scripts around the docker cli itself (which I'm trying to remove but that's another story) and if one day by any chance we decide to change things inside the backends like removing docker or this kind of stuff (won't happen but still) well in this case your stage will break.

What I would recommend here is execute this stage on worker of type `kube_pod` with docker cli installed and bind the docker socket. (the same way that we do for zenko or backbeat, if I remember well).
Now I know that's not a good solution in the long term, and I'll work on that.

As per the `--no-cache` I think it's good to leave it there, everybody have some cache layers of cloudserver hanging and their computer, it's good to have something that build it from scratch :smile:

Reply:
I think you're right, maybe the `kube_pod` would be a better approach and backbeat is already doing it this way. 

Thanks for the input!